Documentation/Build/index.rst

```diff
@@ -226,19 +226,6 @@ Paraview
   Paraview_
 
 
-QGoImageCompare
```

Comment:
Maybe we should mention [ITK-SNAP](http://www.itksnap.org/) here? If you open multiple instances of it, it synchronizes cursors positions and camera views. It is built on ITK and VTK.

Reply:
It is mentioned above